src/module.c

```diff
@@ -98,8 +103,10 @@ static void call_js_event_handler(napi_env env, napi_value js_callback, void* co
     napi_throw_error(env, NULL, "Unable to get global");
   }
 
-  const int argc = 3;
-  napi_value argv[argc];
+#define CALL_JS_CALLBACK_ARGC 3
```

Comment:
`cl.exe` on windows complains if we don't use `#define` for constants used during compile time.